reservia.html

```diff
@@ -2,44 +2,289 @@
 <html>
     <head>
         <meta charset="utf-8">
-        <link rel="stylesheet" href="reservia.css">
-        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
         <title>Reservia</title>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+        <link rel="stylesheet" href="fontface.css">
+        <link rel="stylesheet" href="header.css">
+        <link rel="stylesheet" href="reservia.css">
```

Comment:
should not be used because you splitted the code one week ago for readability purpose.